src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -13,23 +13,29 @@
       enableMenuToggle: true
     });
 
-    function hideArrow() {
+    function hideArrow(br) {
+      if (!br.refs || !br.refs.$BRnav) {
+        return;
+      }
       var $menuTab = br.refs.$BRnav.children('.js-tooltip');
       $menuTab.css('display', 'none');
     }
 
     function setupNavForToggle(br) {
-      hideArrow();
+      hideArrow(br);
       registerEventHandlers(br);
     }
 
     function alwaysShowNav(br) {
-      hideArrow();
+      hideArrow(br);
       removeToggleFromNav();
       br.showNavigation();
     }
   
     var removeToggleFromNav = function removeToggleFromNav(br) {
+      if (!br.refs || !br.refs.$BRnav) {
+        return;
+      }
       var $menuTab = br.refs.$BRnav.children('.js-tooltip');
```

Comment:
And, this looks like a complex object... 
**Question**: should this be documented in the header? Where to find the object definition?

Reply:
good notion, i just added a comment at the top of the file to note this.